piyushgarg-dev/piyushgargdev-nextjs
# fix: Issue #1068

## What does this PR do?

This pull request fixes the issue 1068 , now the contents in the Services component are visible in mobile screens

Fixes #1068 

![Screenshot (181)](https://github.com/piyushgarg-dev/piyushgargdev-nextjs/assets/99383896/5a2589c5-cbb0-471d-b9ed-cad529c1ec1c)


## Type of change

Changed the stylings

## How should this be tested?

1. Go to 'inspect mode'
2. Switch mobile view
3. Scroll down to 'Services'
4. See error
